models/pipeline.js

```diff
@@ -62,7 +67,7 @@ module.exports = {
      * @type {Joi}
      */
     get: Joi.object(mutate(MODEL, [
-        'id', 'scmUri', 'createTime', 'admins'
+        'id', 'scmUri', 'scmContext', 'createTime', 'admins'
```

Comment:
We plans to send a pull-request for api which requires the user schema has `scmContext`.
So, in users schema, `scmContext` is required.

To migrate these tables, I submit the comment to the issue.
https://github.com/screwdriver-cd/screwdriver/issues/365#issuecomment-313570292